src/transforms/groupby.js

```diff
@@ -35,6 +35,35 @@ exports.attributes = {
             'with `x` [1, 3] and one trace with `x` [2, 4].'
         ].join(' ')
     },
+    nameformat: {
```

Comment:
Personally, I like `%`. Though maybe I'd vote for enforcing it with `{}` e.g. `%{name}` similar to ES6 `${name}`.